test/integration/dss/test_dss_api.py

```diff
@@ -21,6 +21,18 @@
 from test import reset_tweak_changes, TEST_DIR
 
 
+def get_uploaded_file_names(path: str):
+    files = []
+    for entry in os.scandir(path=path):
+        if entry.is_file():
+            files.append(entry.name)
+        if entry.is_dir():
```

Comment:
```suggestion
        elif entry.is_dir():
```

and `else: assert False` at the end.